src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -2538,6 +2538,6 @@ def crd_cleanup_force_delete(kubectl_client_location, kube_config, kube_context)
 
 def check_operation_support(operation_name, agent_version):
     error_summary = 'This CLI version does not support {} for Agents older than v1.14'.format(operation_name)
-    if (version.parse(agent_version) < version.parse("1.14.0")):
+    if '-' not in agent_version and (version.parse(agent_version) < version.parse("1.14.0")):
```

Comment:
Add a comment here explaining why we are doing this.  Something like "We want to do this check only for Stable versions, which will be of the pattern '1.x.x' versus other internal versions e.g. '1.x.x-dev'"